psydac/api/equation.py

```diff
@@ -161,7 +161,7 @@ def __init__(self, expr, *args, **kwargs):
         # ...
         # Create boundary equation (None if not needed)
         eqn_bc   = l2_boundary_projection(expr)
-        eqn_bc_h = DiscreteEquation(eqn_bc, domain, [trial_space, trial_space]) \
+        eqn_bc_h = DiscreteEquation(eqn_bc, domain, [trial_space, trial_space], **kwargs) \
```

Comment:
Good catch! Please mention this bugfix in the PR description

Reply:
done